flink-connector-redis/src/main/java/org/apache/flink/streaming/connectors/redis/descriptor/RedisValidator.java

```diff
@@ -30,6 +30,7 @@
     public static final String REDIS_MASTER_NAME = "master.name";
     public static final String SENTINELS_INFO = "sentinels.info";
     public static final String SENTINELS_PASSWORD = "sentinels.password";
+    public static final String CLUSTER_PASSWORD = "cluster.password";
```

Comment:
Could we follow the same pattern for the variables (e.g. `REDIS_CLUSTER_PASSWORD`)